hummingbot/core/utils/async_utils.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
+import logging
+
+
+def asyncio_ensure_future(*args, **kwargs):
+    try:
+        return asyncio.ensure_future(*args, **kwargs)
+    except Exception as e:
+        logging.getLogger().error(f"Unhandled error in background task: {str(e)}", exc_info=True)
+
+
+async def asyncio_gather(*args, **kwargs):
```

Comment:
The wrapped gather function should still raise errors s.t. outer layers can catch them again.

So, right after the debug log, there should be a `raise` to forward the error to outside.